### ISSUE TITLE
Will Material3D dataset be released?

### ISSUE THREAD

**Issue by snowflakewang** [3DTopia/MaterialAnything#1]
Hello, thank you for your great work in the automatic PBR generation for untextured 3D meshes! I am curious about your filtered high-quality PBR dataset from the Objaverse. Do you plan to release it in the future?
Thank you! :)

**Comment by wqdong8** [comment]
+1

**Comment by xhuangcv** [comment]
Yes, we plan to release the object uids and the blender script for rendering multi-view materials.

**Comment by xhuangcv** [comment]
Hi guys, the object uids and blender scripts are available！
